config/config.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
```

Comment:
Let's just use `os.Getenv` for now. 
We can set default values for addr.

If required we'll add support for config. Let's start with barebone.